mesh/accountSettings/views.py

```diff
@@ -29,3 +29,25 @@ def display_theme(request):
                 response.update({"status": "error"})
                 response.update({"message": "An account does not exist with this account ID."})
                 return JsonResponse(response)
+
+class SettingsView(View):
+    def get(self, request, *args, **kwargs):
+        """
+        Handles GET requests when the client fetches for all accounts settings
+        """
+        settings = Settings.objects.all()
+        accounts_settings = serializers.serialize('json', settings)
```

Comment:
Maybe it's just me, but I somewhat dislike how Django's built-in serializer serializes the data, as even on my task which is very similar to this, using the built-in serializer returns just a not so pretty looking string.

For example, when I test this end-point, the output I get looks like this:
```'[{"model": "accountSettings.settings", "pk": 1, "fields": {"isVerified": true, "verificationToken": "1", "hasContentFilterEnabled": true, "displayTheme": "1", "is2FAEnabled": true}}]'```

but if I change the serializer code to be more like this (similar to how I went about this "issue" in my task):

```
    accounts_settings = [
            {
                "accountID": setting.accountID.accountID,
                "isVerified": setting.isVerified,
                "VerificationToken": setting.verificationToken,
                "HasContentFilterEnabled": setting.hasContentFilterEnabled,
                "DisplayTheme": setting.displayTheme,
                "Is2FAEnabled": setting.is2FAEnabled
            }
            for setting in settings
        ]
```

it will output like this:
```
[{"accountID": 1, "isVerified": true, "VerificationToken": "1", "HasContentFilterEnabled": true, "DisplayTheme": "1", "Is2FAEnabled": true}]
``` 

which is much prettier to look at if you ask me.

I have some examples of what it looks like in a thread about this specific B5.3 issue if you want to see screenshots of what it actually looks like, but currently I'm waiting for Bob's response or even Chris' response if he reads this before Bob.

For the time being I think the way you have it is fine and can stay that way, but I'm going to hold off on approving until I can get Bob or Chris' opinion.

Other than that though, good work 👍🏼

Reply:
To clarify I don't necessarily enjoy this solution as much cause it is somewhat manual and to do this for every model would be painstaking, but I'm open to other ideas 👍🏼

Reply:
Update from Chris is that it is probably fine to use Django Serializer, so I think what you have is good!

Reply:
👍